lib/datasource/deb/index.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { copyFile, mkdirp, remove, stat } from 'fs-extra';
+import { getPkgReleases } from '..';
+import type { GetPkgReleasesConfig } from '..';
+import * as httpMock from '../../../test/http-mock';
+import { GlobalConfig } from '../../config/global';
+import { DebDatasource } from '.';
+
+describe('datasource/deb/index', () => {
+  describe('getReleases', () => {
+    const testPackagesFile = __dirname + '/test-data/Packages.gz';
+    const extractedTestFile = __dirname + '/test-data/Packages';
```

Comment:
We have helpers for this, checkout fixtures class

Reply:
The only test case where I think a fixture would make sense is [this one](https://github.com/Ka0o0/renovate/blob/feat/deb-datasource/lib/datasource/deb/index.spec.ts#L44). However, I don't quite know how to get it running.

If I understood it correctly, the Fixtures.mock method will register some temp files based on what is provided as a parameter and intercept FS calls to those files? So I'm setting up the test case like this (basically replacing line 44 - 45):

```JavaScript
const fixtures: DirectoryJSON = {};
fixtures[extractedPackageFile] = Fixtures.get('Packages');
Fixtures.mock(fixtures);
```
But now the test case would fail [here](https://github.com/Ka0o0/renovate/blob/feat/deb-datasource/lib/datasource/deb/index.spec.ts#L46). Shouldn't the Fixtures' mock make sure that the file exists?